services/node/package.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "reviews",

```

```suggestion
  "name": "node",

```

This needs to be unique for lerna bootstrapping to not fail